src/Forms/Main.cs

```diff
@@ -7740,8 +7740,8 @@ private void SplitSelectedParagraph(double? splitSeconds, int? textIndex)
 
                 if (currentParagraph.StartTime.IsMaxTime && currentParagraph.EndTime.IsMaxTime)
                 {
-                    newParagraph.StartTime = TimeCode.MaxTime;
-                    newParagraph.EndTime = TimeCode.MaxTime;

```

Comment:
If `newParagraph.StartTime` or `newParagraph.EndTime` changes wit will cause the `TimeCode.MaxTime` to change aswell.